### ISSUE TITLE
Finding accuracy using different ML algo

### ISSUE THREAD

**Issue by Vidip-Ghosh** [Ayushpanditmoto/Trading-Bot#58]
I can try to find accuracy using different ML algorithms like Linear Regression, Decision Tree, Random forest and considering that algorithm having highest accuracy.

**Comment by Ayushpanditmoto** [comment]
I will add label later

**Comment by Tanmaymathur03** [comment]
I am going to explore different algorithms and implement whichever provides best accuracy.

**Comment by Vidip-Ghosh** [comment]
<img width="1333" alt="Screenshot 2024-05-19 at 23 12 28" src="https://github.com/Ayushpanditmoto/Trading-Bot/assets/91741581/5b9ea5b2-eec4-46ce-bd21-358f2d694c48">
@Ayushpanditmoto I tried finding accuracy using Linear regression & Random forest. I was reading about R2 Score, if r2 score is 0.99 then model may perform well. 
Its showing 99 since I multiplied R2_Score*100. What are your views on it?
